src/storage-controller/src/instance.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) struct Instance<T> {
     /// A function that returns the current time.
     now: NowFn,
     /// A sender for responses from replicas.
```

Comment:
Could you extend this comment to say what the meaning of the `Option<ReplicaId>` is, especially if it's `None`?